packages/map-components/templates/vue/src/App.vue

```diff
@@ -1,6 +1,36 @@
+<script setup lang="ts">
+import { defineCustomElements } from '@arcgis/map-components/dist/loader'
+defineCustomElements(window)
+
+function onReady(event: any) {
+  console.log("Map View ready", event);
+}
+
+const popupOptions = defineProps({
+  dockEnabled: {
+    type: Boolean,
+    default: () => true
+  },
+  dockOptions: {
+    type: Object,
+    default: () => {
+      return {
+        position: 'bottom-right',
+        breakpoint: false
+      }
+    }
+  }
+})
```

Comment:
Great example on how to set properties on a map component in Vue! However, if we're doing this in one template then it might be best to do this in all templates for consistency?

Reply:
removed it for now